cmd/logs.go

```diff
@@ -608,10 +603,11 @@ func buildQueries(numberOfRequests int, r rng, sources []string, terms []term) [
 
 func (response logResponse) String() string {
 	output := make([]string, 0)
-	for _, hit := range response.Hits.Hits {
-		source := hit.Source
-		output = append(output, fmt.Sprintf("%v %v %v | %v", source.Timestamp, source.Severity, source.Source, source.Log))
+	hits := response.Hits.Hits
+	for i := len(hits) - 1; i >= 0; i-- {
+		source := hits[i].Source
+		output = append(output, fmt.Sprintf("%v %v %v\t| %v", source.Timestamp, source.Severity, source.Source, source.Log))
```

Comment:
What do you think about to use this format ?

```suggestion
		output = append(output, fmt.Sprintf("%v\t%v\t%v\t|\t%v", source.Timestamp, source.Severity, source.Source, source.Log))
```

Reply:
According to the [documentation](https://golang.org/pkg/text/tabwriter/#Writer) 
>The Writer treats incoming bytes as UTF-8-encoded text consisting of cells terminated by horizontal ('\t') or vertical ('\v') tabs

So please, use the formatting that I proposed. It will align the output in four columns